### DISCUSSION TITLE
How to use doctr in Android and iOS?

### DISCUSSION THREAD

**Discussion by yuvaraj119** [mindee/doctr#1748] [Q&A]
### Bug description

How to use doctr in Android and iOS?

### Code snippet to reproduce the bug

How to use doctr in Android and iOS?

### Error traceback

How to use doctr in Android and iOS?

### Environment

How to use doctr in Android and iOS?

### Deep Learning backend

How to use doctr in Android and iOS?

**Comment by felixdittrich92** [comment]
#1726
